code/src/components/CustomCheckbox.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+// import { useDispatch } from "react-redux";
```

Comment:
maybe this comment can be removed if useDispatch isn't used?

Reply:
Done!